packages/disputer/.mocharc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  timeout: 100000,
```

Comment:
why?

Reply:
default mocha time is too slow for some of the our tests.

Reply:
Yeah, agreed. We could probably set this to something lower than 100s, but we set it to 100s elsewhere, and it shouldn't change performance unless we have a timeout.